config/config.go

```diff
@@ -106,17 +106,18 @@ type File struct {
 }
 
 type Consul struct {
-	Addr          string
-	Scheme        string
-	Token         string
-	KVPath        string
-	TagPrefix     string
-	Register      bool
-	ServiceAddr   string
-	ServiceName   string
-	ServiceTags   []string
-	ServiceStatus []string
-	CheckInterval time.Duration
-	CheckTimeout  time.Duration
-	CheckScheme   string
+	Addr                 string
+	Scheme               string
+	Token                string
+	KVPath               string
+	TagPrefix            string
+	Register             bool
+	ServiceAddr          string
+	ServiceName          string
+	ServiceTLSSkipVerify bool
```

Comment:
This should be `CheckTLSSkipVerify` and pls move it to the bottom below `CheckScheme`